src/client/components/ForumPost.js

```diff
@@ -32,6 +30,38 @@ function ForumPost(props) {
     if (!showModal) setShowModal(true);
   }
 
+  function queryComments() {
+    fetch('http://localhost:5000/api/forum/post/comments/postID')
+      .then(res => res.json())
+      .then(
+        (serverResult) => {
+          JSON.stringify();
+          setComments(serverResult);
+          setLoaded(true);
+        }
+      );
+  }
+
+  function handleCommentSubmit() {
+    const request = {
+      method: 'POST',
+      mode: 'same-origin',
+      headers: { 'Content-Type': 'application/json' },
+      body: {
+        body: document.getElementById('new-comment-field'),
+      }
+    };
+    fetch('http://localhost:5000/api/forum/create/comment', request);
```

Comment:
For `fetch` method call, I suggest we handle the callback, be it response or error.

Reply:
Ok fixed